source/modules/MEI.cmn.xml

```diff
@@ -291,9 +291,12 @@
         </datatype>
       </attDef>
       <attDef ident="slope" usage="opt">
-        <desc xml:lang="en">Records the slope of the beam.</desc>
+        <desc xml:lang="en">Records the slope of a beam; e.g., in terms of staff interline distance; 
```

Comment:
We should refer to these as virtual units (`vu`) instead of re-explaining what a VU is in MEI.